docs/src/cirrus/components/workflows/batch.rst

```diff
@@ -21,7 +22,7 @@ Similarly, Batch does not integrate returned payloads with step functions nearly
 as well as Lambda, so Cirrus has a built-in ``post-batch`` task to help with,
 well, post-batch operations. Specifically, ``post-batch`` can pull the Batch
 payload from S3 and return that in the case of a successful Batch execution. In
-the case of a Batch error, ``post-batch`` will scrape the execution logs for an
+the case of a Batch error, ``post-batch`` will scrape the batch logs for an
```

Comment:
I perceive this to be an inaccurate change. It scrapes the logs from the execution, the container logs. It does not specifically handle batch errors, and saying "batch logs" makes me think it is looking at something batch-specific, which is really not the case.